src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinEndpoint.cs

```diff
@@ -13,16 +13,76 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+using System.Text.Json;
+
 namespace OpenTelemetry.Exporter.Zipkin.Implementation
 {
     internal class ZipkinEndpoint
```

Comment:
not a struct?

Reply:
I left alone because `ZipkinConversionExtensions` is already maintaining the `ZipkinEndpoint`s it has seen in a cache:
```
        private static readonly ConcurrentDictionary<string, ZipkinEndpoint> LocalEndpointCache = new ConcurrentDictionary<string, ZipkinEndpoint>();
        private static readonly ConcurrentDictionary<string, ZipkinEndpoint> RemoteEndpointCache = new ConcurrentDictionary<string, ZipkinEndpoint>();
```